### DISCUSSION TITLE
Analog Temp on Wemos D1

### DISCUSSION THREAD

**Discussion by lalo-uy** [arendst/Tasmota#18292] [Q&A]
As states in the Analog driver there are 2 posible configuration :  

Mode 0 : Shelly 2.5 NTC Thermistor
3V3 --- ANALOG_NTC_BRIDGE_RESISTANCE ---v--- NTC --- Gnd
                                                                                 |
                                                                             ADC0
Mode 1 : NTC towards 3V3 (Sinilink Thermostat Relay Board (XY-WFT1)
 3V3 --- NTC ---v--- ANALOG_NTC_BRIDGE_RESISTANCE --- Gnd
                         |
                   ADC0

But Wemos D1 have a divisor in the analog pin, so the circuit is:
 3V3 --- ANALOG_NTC_BRIDGE_RESISTANCE ---v--- NTC --- Gnd
                                         |
                                       220K
                                         | _____ 100K______Gnd
                                        |
                                        ADC0

In that situation line 533      
   Rt = (double)Adc[idx].param1 * (double)adc / (ANALOG_RANGE * ANALOG_V33 - (double)adc);

need to be:       
 Rt = (double)Adc[idx].param1 * (double)adc / (ANALOG_RANGE  - (double)adc);

as in ESP32.
I tested on a personal firmware and works OK

**Comment by Duvio** [comment]
I just encountered the same problem. Would you mind creating a corresponding issue?

**Comment by lalo-uy** [comment]
I don’t know how to make a PR. And a new define has to be created just for WEMOSD1
